src/api/app/controllers/webui/project_controller.rb

```diff
@@ -665,15 +665,19 @@ def meta
   end
 
   def save_meta
+    request_data = Xmlhash.parse(params[:meta])
+    params[:user] = User.current.login
+
+    @project = Project.get_by_name(params[:project])
```

Comment:
Because the before filter uses ``Project.find_by_name`` and the API implementation uses ``Project.get_by_name`` (self implemented SQL ...). Before we used the project from the before filter only for authentication and not for the "logic" (because the API selected the project again by ``Project.get_by_name``. I just want to avoid conflicts here ...